demos/simple_jump_bot/README.md

```diff
@@ -0,0 +1,22 @@
+# HammettSimpleJumpBot
```

Comment:
`requirements.txt` is missing. Add it to make the installation process universal for all future demos.